internal/pkg/gateway/server/apiutils_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2020 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package server
+
+import (
+	"testing"
+
+	"github.com/hyperledger/fabric/protoutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetValueFromResponse(t *testing.T) {
+	response := createProposalResponse("MyResult", t)
+
+	result, err := getValueFromResponse(response)
+	require.NoError(t, err, "Failed to extract value from response")
+	require.Equal(t, []byte("MyResult"), result.Value, "Incorrect value")
```

Comment:
this is better. At some point you're going to need to check the result more completely. When that happens, you're going to want to use `proto.Equal` to compare.